_src/pattern-library/sass/components/_grid.scss

```diff
@@ -0,0 +1,1125 @@
+// ------------------------------
+// edX Pattern Library: Components - Grid
+
+// About: Contains base settings and options for the default Pattern Library grid system.
+// ----------------------------
+
+// #CONFIG
+// #GLOBALS
+// #UTILITY
+// #CANNED
+// #CANNED - RESPONSIVE
+// #GALLERY
+// #GALLERY - CANNED
+// #GALLERY - CANNED + RESPONSIVE
+// #DEBUG
+
+// ----------------------------
+
+// ----------------------------
+// #CONFIG
+// ----------------------------
+
+// susy-based configuration (see http://susydocs.oddbird.net/en/latest/settings/)
+$susy: (
+    use-custom: (
+        background-options: false,
+        box-sizing: false,
+        breakpoint: true,
+        clearfix: true,
+        rem: true
+    ),
+    math: fluid,
+    output: float
+);
+
+// LAYOUT: set default susy settings
+// (gutters are handled outside of elements - by default)
+$susy-default: (
```

Comment:
NOTE: Default grid settings for edX UX, can be changed on a project-by-project basis.